bgpd/bgp_evpn.c

```diff
@@ -6612,6 +6582,63 @@ int bgp_evpn_unimport_route(struct bgp *bgp, afi_t afi, safi_t safi,
 	return install_uninstall_evpn_route(bgp, afi, safi, p, pi, 0);
 }
 
+/*
+ * Export IPv[46] unicast route from VRF to global table
+ */
+int bgp_evpn_export_type5_route(struct bgp *bgp, struct bgp_dest *dest, struct bgp_path_info *pi,
```

Comment:
We never check for the returned value of this function, can we use void?

Reply:
Definitely. I also changed `bgp_evpn_unexport_type5_route` to use void too.